### PR TITLE
Start Front design

### LINKED ISSUE #3: Add static home page
### Type of Issue

- [ ] Bug
- [X] Feature Request
- [ ] Documentation
- [ ] Other

### Description

For now, we will just remake the current theme.
We can't just copy the old HTML because we will be using Vue + HTML + tailwind and not a django template.

### PR DESCRIPTION
**Pull request by KwikKill**
## Description

This PR add a lot of things :
- Improve back consistency
- Add first front design
- Add table component
- Add management tab
- Handle login, logout in both front and back
- Handle User, Device management in both front and back
- Handle database wipe (front side)
- Handle different login case :
  - Known Device
  - New device
  - Known Device but different user
  - ...

## Checklist

- [X] I have tested the changes locally and they work as expected.
- [X] I have tested the responsiveness of the changes and they work as expected.
- [X] I have assigned the pull request to the appropriate reviewer(s).
- [X] I have added labels to the pull request, if necessary.

## Related Issues

Fix #3, #4, #5, #6, #7, #8

### CODE REVIEW

**Review by TheBloodMan49 — APPROVED** [review]
Left my comments IRL, it seems to be working as intended now so LGTM

File: backend/langate/network/apps.py [hunk 1]
```diff
@@ -65,3 +64,26 @@ def ready(self):
                     mark_res = netcontrol.query("set_mark", {"mac": dev.mac, "mark": 100})
                     if not mark_res["success"]:
                         logger.info("[PortalConfig] Could not set mark 0 for device %s", dev.name)
+
+            logger.info(_("[PortalConfig] Add default whitelist devices to the ipset"))
+            if os.path.exists("assets/misc/whitelist.txt"):
+                with open("assets/misc/whitelist.txt", "r") as f:
+                    for line in f:
+                        line = line.strip().split("|")
+                        if len(line) == 2:
```
Comment (SkytAsul) [inline backend/langate/network/apps.py:73]: Du coup on peut pas configurer la mark des appareils whitelist ?

Comment (KwikKill) [reply backend/langate/network/apps.py:73]: Everything related to Marks was delayed to #23 which is still in progress.

**Review by SkytAsul — APPROVED** [review]
Only checked the backend- seems lgtm

**Review by pixup1 — COMMENTED** [review]
Indentation sizes seem to be inconsistent, why did you switch back to 2 spaces ? Some files like netcontrol.py even have a mix of the two widths (💀)

**Comment by pixup1** [comment]
Also a little ergonomy nitpick : When you add an invalid user (i.e. password too short) and press validate, it closes the input prompt and shows a notification. It would probably be better to keep the prompt open so that you can directly correct the mistake instead of having to type everything again.

**Pull request merged**